uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-request-popup-details-tab.component.spec.js

```diff
@@ -27,8 +27,7 @@ define([
     var date2017To = '02/02/2017'; // Must be greater than `date2017`
     var date2017ToInServerFormat = '2017-02-02'; // Must match the date of `date2017To`
 
-    beforeEach(module('common.mocks', 'leave-absences.templates',
-    'leave-absences.mocks', 'manager-leave', function (_$provide_) {
+    beforeEach(module('common.mocks', 'leave-absences.templates', 'leave-absences.mocks', 'manager-leave', function (_$provide_) {
```

Comment:
Question: the new semistandard doesn't like to split parameters in multiple lines?

Reply:
Yes, it messes with indentation.

![semi](https://user-images.githubusercontent.com/3973243/35728645-218daa9c-0804-11e8-9d5c-27956a508514.gif)